compute_tools/src/bin/compute_ctl.rs

```diff
@@ -249,11 +250,20 @@ fn main() -> Result<()> {
 
     state.status = ComputeStatus::Init;
     compute.state_changed.notify_all();
+    let pspec = state.pspec.as_ref().expect("spec must be set");
+    let ttid = TenantTimelineId {
+        tenant_id: pspec.tenant_id,
+        timeline_id: pspec.timeline_id,
+    };
```

Comment:
Why this instead of `tenant_id=`, `timeline_id=` as we use in pageserver?

Reply:
Yes please, should use the two labels `tenant_id=` and `timeline_id=` convention.

Reply:
I dislike that these prefixes occupy half of screen, being useful only for searching, so want them to be as short as possible. Technically I doubt we ever want to search by tenant_id, timeline_id would be enough, but include it for completeness / uniformity.

Reply:
Well, I don't insist much, though note that both safekeepers (and now postgres) also logs ttids.